invenio_app_ils/config.py

```diff
@@ -777,6 +777,9 @@ def _(x):
 
 # RECORDS REST facets
 # =========================
+#: Number of records to fetch by default
+RECORDS_REST_DEFAULT_RESULTS_SIZE = 10
```

Comment:
We don't need this because it is already defined in `invenio-records-rest`, unless we change the value

Reply:
We need it if we dont want to retrieve it from `current_app.wsgi_app.mounts['/api'].config`!

Reply:
Yes, we discussed it and decided to keep it so we don't have to do that. But it would be nice to have all config in one place. :)

Reply:
You could expose the whole api config in the current_app['api_config'] probably :)